connector/Makefile

```diff
@@ -90,8 +84,8 @@ develop:
 	@echo "Building and installing $(PKG_NAME) development distributable: $@"
 	@echo ""
 
-	@pip uninstall -y yang.connector || true
-	@./setup.py develop -q
+	@pip3 uninstall -y yang.connector || true
+	@$(PYTHON) setup.py develop -q
```

Comment:
why need this change? do we really need to use `pip3`/`python3`? what I can think of is, if you have both python2 and 3 and not using virtualenv, you might want this change.

Reply:
Some environments still default to `python2` , `python` would then run the `python2` interpreter. To be able to run the make commands in such an environment, this change explicitly calls `python3`. This was already changed in most pyats repos, just not this one.